### ISSUE TITLE
[FEAT] Question difficulty counter 

### ISSUE THREAD

**Issue by rahulsagu** [Preclinify/Tickets#29]
(See ticket #28 for further information)

For the backend, we should add a counter that measures the number of first attempts for each question, which is only viewable from the back end. This would be counted from the closed launch onwards. This should be implemented so we can determine if a question has met the minimum required number of first attempts (exact number to be confirmed). Once the threshold has been met and the percentage of correct answers has been determined, the question can be classified accordingly in terms of difficulty (easy, medium or hard difficulties). Until the threshold has been met, the question remains unclassified.

**Comment by Joecey** [comment]
Once 0.0.4 testing is merged into main - please ensure that we have tracking on both exam and endless modes (given the way answers are corrected, the method on how these work may differ)
